scripts/build/libc/glibc.sh

```diff
@@ -352,6 +352,12 @@ do_libc_backend_once() {
             build_cppflags="${build_cppflags} -I${CT_BUILDTOOLS_PREFIX_DIR}/include/"
             build_ldflags="${build_ldflags} -lintl -liconv"
             ;;
+        *freebsd*)
+            # .. and the same goes for FreeBSD. Additionally, FreeBSD's stat is
+            # always 64bit, so replace struct stat64 with stat.
+            build_cppflags="${build_cppflags} -I${CT_BUILDTOOLS_PREFIX_DIR}/include/ -Dstat64=stat"
```

Comment:
Which, as I said, is fine by me - just pass that option for cygwin build machines, too, and retire the respective patches.

The patch was there for years, but somehow it wasn't propagated across all glibc versions. I only did that recently - so this is probably why you encountered that issue.